setup.cfg

```diff
@@ -31,7 +31,7 @@ install_requires =
     nltk~=3.5
     numpy ~=1.21.5; python_version<'3.8'
     numpy ~=1.23.3; python_version>='3.8'
-    pandas~=1.3.5
+    pandas~=1.5.2
```

Comment:
so - i guess this is as good a place as any to ask this - I ran into images on the CUDA image with this vintage of pandas, but I see that that's breaking the 3.7 release. Is it safe to specify a range of 1.3.5-1.5.2?

Reply:
We could do what I did for numpy and specify a different version for 3.7 and 3.8+. What was the issue with 1.3.5?

Reply:
well, it looks like i can't replicate the issue now, so i'll go ahead and reset this. 💻 🔥